chaos.go

```diff
@@ -38,6 +56,15 @@ func Float[F constraints.Float](n F, seed ...any) F {
 	return F(r.Float64()) * n
 }
 
+// FloatInRange returns a deterministic float between min and max.
+func FloatInRange(min, max float64, seed ...any) float64 {
```

Comment:
Functions that can panic should contain the "Must" indicator

Reply:
It means include "Must" in the function name ?